packages/json-schemas/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "5.2.0",
+        "changes": [
+            {
+                "note": "Add `exchangeProxyMetaTransactionSchema`",
+                "pr": 2657
```

Comment:
This PR will eventually be merged into the other, so I'm using the PR # of that one.